scenarios/ecs_privesc_evade_protection/terraform/ecr.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_ecr_repository" "repository" {
+  name = "cg-repository-${var.cgid}"
+}
+
+resource "null_resource" "docker_image" {
+
+  # Push Docker image when the scenario be created
+  provisioner "local-exec" {
+    when       = create
+    on_failure = fail
+    command    = "python ./push-dockerfile.py --dockerfile_path ../assets/ssrf-web/ --repository ${aws_ecr_repository.repository.name} --region ${var.region} --profile ${var.profile} --image_tag latest"
```

Comment:
Switch out `python` to `python3` not all systems come with python/ an alias to 3 (osx)

Python3 is already defined to be used in `cloudgoat.py`

The files could also be made executable and then define the env in the top of the files